package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@financial-times/o-brand",
-  "version": "100.0.0-13",
+  "version": "3.2.9",
```

Comment:
do you want this to have `-beta.1` at the end?

Reply:
That should be handled by `origami-ci-tools` as part of the `release-origami-component` action, I think

Reply:
Well I guess it will in the package because we run `npm version` as part of `origami-ci-tools` but I guess nothing will commit that change.

Reply:
So [committing the version wasn't so straight forward, details in the proposal](https://github.com/Financial-Times/origami/pull/111). For now component releases can continue like normal as there's no obvious need for committing the version to `package.json`, so long as it's correct in the published bundle. It we need to commit the version to source code, for example to expose the version in the component api for debugging or to handle versioned assets, we can revisit.